<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_interfaceClass</class>
 <widget class="QMainWindow" name="main_interfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShut_Current_Window"/>
    <addaction name="actionShut_All_Windows"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/new.png</normaloff>:/main_interface/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/open.png</normaloff>:/main_interface/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/save.png</normaloff>:/main_interface/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/saveAs.png</normaloff>:/main_interface/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/undo.png</normaloff>:/main_interface/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/redo.png</normaloff>:/main_interface/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/cut.png</normaloff>:/main_interface/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/copy.png</normaloff>:/main_interface/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/paste.png</normaloff>:/main_interface/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionShut_Current_Window">
   <property name="text">
    <string>Shut Current Window</string>
   </property>
  </action>
  <action name="actionShut_All_Windows">
   <property name="text">
    <string>Shut All Windows</string>
   </property>
  </action>
  <action name="actionTiled">
   <property name="text">
    <string>Tiled</string>
   </property>
  </action>
  <action name="actionStacked">
   <property name="text">
    <string>Stacked</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/previous.png</normaloff>:/main_interface/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/next.png</normaloff>:/main_interface/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="main_interface.qrc">
     <normaloff>:/main_interface/images/exit.png</normaloff>:/main_interface/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main_interface.qrc"/>
 </resources>
 <connections/>
</ui>
